### PR TITLE
Make the InstructionWriter put_xxx methods return a bool

### PR DESCRIPTION
**Pull request by s1341**
This is more in line with the gum api

### CODE REVIEW

File: frida-gum/src/instruction_writer.rs [hunk 1]
```diff
@@ -438,15 +438,15 @@ impl InstructionWriter for X86InstructionWriter {
         }
     }
 
-    fn put_bytes(&self, bytes: &[u8]) {
+    fn put_bytes(&self, bytes: &[u8]) -> bool {
         println!("put_bytes: {:?}", bytes);
```
Comment (meme) [inline frida-gum/src/instruction_writer.rs:442]: Please remove this `println!`, it slipped through

**Review by meme — CHANGES_REQUESTED** [review]
LGTM, but fix CI/build errors please

**Comment by s1341** [comment]
Done. Good to merge. Sorry for the delay.

File: frida-gum/src/instruction_writer.rs [hunk 2]
```diff
@@ -438,15 +438,15 @@ impl InstructionWriter for X86InstructionWriter {
         }
     }
 
-    fn put_bytes(&self, bytes: &[u8]) {
-        println!("put_bytes: {:?}", bytes);
+    fn put_bytes(&self, bytes: &[u8]) -> bool {
         unsafe {
-            gum_sys::gum_x86_writer_put_bytes(self.writer, bytes.as_ptr(), bytes.len() as u32)
+            gum_sys::gum_x86_writer_put_bytes(self.writer, bytes.as_ptr(), bytes.len() as u32);
+            true
```
Comment (meme) [inline frida-gum/src/instruction_writer.rs:444]: Interesting that x86 does not return a boolean but all other architectures do...

Comment (meme) [inline frida-gum/src/instruction_writer.rs:444]: Interesting that x86 does not return a boolean but all other architectures do...

**Review by meme — APPROVED** [review]

**Pull request merged**